src/M2ulPhyS.hpp

```diff
@@ -276,6 +280,8 @@ class M2ulPhyS : public TPS::Solver {
   // domain extent
   double xmin, ymin, zmin;
   double xmax, ymax, zmax;
+  double local_xmin, local_ymin, local_zmin;
```

Comment:
Is there a reason these are kept as class members as opposed to just having them be local variables within the `initVariables()` function?